tests/test_grid3d/test_grid_xtgformats_io.py

```diff
@@ -345,3 +346,40 @@ def read():
     benchmark(read)
 
     assert benchmark_gridprop.values[0:2, 0:2, :].all() == prop2.values.all()
+
+
+def test_hdf5_partial_import(benchmark_grid, tmp_path):
+    fname = join(tmp_path, "reek_geo_grid.compressed_h5")
+
+    benchmark_grid._zcornsv += 1.0
+
+    fna = benchmark_grid.to_hdf(fname, compression="blosc")
+
+    grd2 = xtgeo.Grid()
+
+    grd2.from_hdf(fna, ijkrange=(1, 3, 1, 5, 1, 4))
```

Comment:
Just out of curiosity, does the partial import allow for random selection? (the test seems to only check when indexes start at 1). And if so, could we test that as well?

Reply:
I guess we could try for other indecies than `1` at least. The test data was selected as to be a regression test for the bug.

Reply:
Added a test with some generation of random ijk ranges